curl-and-grep/curl-and-grep.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: curl-and-grep
+spec:
+  inputs:
+    params:
+    - name: url
+      description: URL to be curl'ed.
+    - name: text
+      description: text to grep
+      default: "HTTP"
+    - name: buffer
```

Comment:
What is the reason for the buffer? You could `curl -O- URL | grep TEXT`.

Since the image is the same there is not much overhead in using two steps, but even with the buffer this could be easily done as a single step `curl -O[BUFFER] [URL]; grep [TEXT] [BUFFER]`.

Reply:
good idea, let me give it a try.